common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -90,9 +90,10 @@ enum eGlobals {
     PHY_RATE_100KB_MAX                      = 8666,
     PHY_RATE_100KB_MIN                      = 72,
     PHY_RATE_100KB_INVALID                  = 0,
-    BSS_STEER_DISASSOC_TIMER_MS             = 200, // ~200ms
-    BSS_STEER_IMMINENT_VALID_INT            = 100,
-    BSS_STEER_VALID_INT                     = 50,
+    BEACON_TX_TIME_MS                       = 100,
+    BSS_STEER_DISASSOC_TIMER_MS             = 2000,
+    BSS_STEER_IMMINENT_VALID_INT            = (BSS_STEER_DISASSOC_TIMER_MS / BEACON_TX_TIME_MS),
+    BSS_STEER_VALID_INT                     = 2, // 200ms
```

Comment:
That sounds *awfully* short as a validity time... I would actually expect it to be higher in case we don't disassociate - although really, why would the validity time be different in either case?

Reply:
validity interval of 200ms was not enough for imminent steering, most of the times we would just disconnect the client before it even answers.
but , for non imminent steering requests client usually responds right away (maybe he doesn't have to perform a neighbor scan ) .
Also , for imminent bss requests cases i wanted it to be depended on the dissociation time in case someone changes it in the future and forget to update the validity interval as well.